.travis.yml

```diff
@@ -1,8 +1,6 @@
 language: rust
 cache: cargo
 rust:
-  - stable
-  - beta
   - nightly
```

Comment:
We should probably also test against a pinned nightly version too, so we have a known-good point in time to compare with when nightly changes.